remote.pro

```diff
@@ -506,6 +506,11 @@ win32 {
 
 DISTFILES += \
     components/FpsItem.qml \
+    components/media_player/ui/CardMediaPlayerEPGView.qml \
+    components/media_player/ui/CardMediaPlayerEPGView.qml \
+    components/media_player/ui/CardMediaPlayerRemote.qml \
+    components/media_player/ui/EPGItemView.qml \
+    components/media_player/ui/MediaPlayerRemoteView.qml \
```

Comment:
These should not be added here. These are in a resource file.

Reply:
solved